README.md

```diff
@@ -44,6 +44,9 @@ The following table lists the software versions NGINX Kubernetes Gateway support
 | NGINX Kubernetes Gateway | Gateway API | Kubernetes | NGINX OSS |
 |-|-|-|-|
 | 0.1.0 | 0.5.0 | 1.19+ | 1.21.3|
+| 0.2.0 | 0.5.0 | 1.21+ | 1.21.x *|
```

Comment:
Do we want to put the newest release at the top? That's what we do in other projects, but I'm fine either way.